posts/urls.py

```diff
@@ -4,7 +4,6 @@
 urlpatterns = [
     path('create/', CreatePostView.as_view(), name="create-post"),
     path('delete/', DeletePostView.as_view(), name="delete-post"),
-    path('view/<int:pk>/',PostDetailView.as_view(),name="view post")
+    path('view/', PostDetailView.as_view(), name="view-post"),
```

Comment:
How exactly is this going to display a post if you are not sending any post id

Reply:
pk is passed as a GET request parameter